operator/pkg/central/extensions/reconcile_tls.go

```diff
@@ -149,6 +160,27 @@ func (r *createCentralTLSExtensionRun) generateCentralTLSData() (types.SecretDat
 	return fileMap, nil
 }
 
+func (r *createCentralTLSExtensionRun) reconcileCentralDBTLSSecret(ctx context.Context) error {
+	if r.centralObj.Spec.Central.IsExternalDB() {
```

Comment:
Can we flip the conditional so it's consistent with those below it (if then reconcile else delete)

Reply:
I put it this way to keep the condition negation-free...

Reply:
I will take a look at renaming `IsExternalDB` though :-)

Reply:
Tracking in https://issues.redhat.com/browse/ROX-21608